app/views/models/bulk_edit.html.erb

```diff
@@ -52,12 +52,12 @@
   <div class="row mb-3">
     <%= form.label :new_library_id, class: "col-sm-2 col-form-label" %>
     <div class="col-sm-10">
-      <%= form.collection_select :new_library_id, Library.all, :id, :name, {selected: @library.id, include_blank: true}, {class: "form-control col-auto form-select"} %>
+      <%= form.collection_select :new_library_id, Library.all, :id, :name, {include_blank: true}, {class: "form-control col-auto form-select"} %>
     </div>
   </div>
 
-  <%= render 'tags_edit', :form => form, :name => :add_tags, :value => "", :label => "Add Tags", tags: @library.all_tags %>
-  <%= render 'tags_edit', :form => form, :name => :remove_tags, :value => "", :label => "Remove Tags", tags: [] %>
+  <%= render 'tags_edit', :form => form, :name => :add_tags, :value => "", :label => "Add Tags", tags:  @addtags?@addtags:[] %>
```

Comment:
The ternary here can be a bit more concise in Ruby:

```suggestion
  <%= render 'tags_edit', :form => form, :name => :add_tags, :value => "", :label => "Add Tags", tags:  @addtags || [] %>
```

Reply:
nice :)

Reply:
Yeah, gotta love Ruby for readability :)